src/pages/student/components/AdministrativeInfo/AdministrativeInfo.tsx

```diff
@@ -46,30 +46,37 @@ export default function AdministrativeInfo(props: VisualComponent & Administrati
 					cells={materialCells}
 					onChange={({ data }): void => {
 						onChange(data);
+						console.log(Models.ScholarshipType.Bidding.valueOf());
+						console.log(data);
+						console.log(student.administrative_info.scholarship_type);
 					}}
 					readonly={!editable}
 					ajv={handleDefaultsAjv}
 				/>
 
-				<FormControl variant="standard" sx={{ width: "100%" }}>
-					<InputLabel id="agreement-type-label">Convenio</InputLabel>
+				{student.administrative_info.scholarship_type == Models.ScholarshipType.Agreement ||
+				student.administrative_info.scholarship_type == Models.ScholarshipType.Bidding.valueOf() ? (
+					<FormControl variant="standard" sx={{ width: "100%" }}>
+						<InputLabel id="agreement-type-label">Convenio</InputLabel>
 
-					<Select
-						labelId="agreement-type"
-						id="agreement-type"
-						label="Convenio"
-						value={agreementType}
-						disabled={!editable} onChange={(event) => setAgreementType(event.target.value)}>
-						{dataStore.agreementTypes &&
-							dataStore.agreementTypes?.map((value, index) => {
-								return (
-									<MenuItem key={index} value={value}>
-										{value}
-									</MenuItem>
-								);
-							})}
-					</Select>
-				</FormControl>
+						<Select
+							labelId="agreement-type"
+							id="agreement-type"
+							label="Convenio"
+							value={agreementType}
+							disabled={!editable}
+							onChange={(event) => setAgreementType(event.target.value)}>
+							{dataStore.agreementTypes &&
+								dataStore.agreementTypes?.map((value, index) => {
```

Comment:
```suggestion
								dataStore.agreementTypes.map((value, index) => {
```
Entiendo que no se necesita el '?' porque lo estás preguntando antes, no?